prepared.go

```diff
@@ -81,8 +61,16 @@ func NewPrepareRequest(expr string) *PrepareRequest {
 }
 
 // Body fills an msgpack.Encoder with the execute request body.
-func (req *PrepareRequest) Body(res SchemaResolver, enc *msgpack.Encoder) error {
-	return fillPrepare(enc, req.expr)
+func (req *PrepareRequest) Body(_ SchemaResolver, enc *msgpack.Encoder) error {
+	err := enc.EncodeMapLen(1)
+	if err != nil {
+		return err
+	}
+	err = enc.EncodeUint(uint64(iproto.IPROTO_SQL_TEXT))
+	if err != nil {
+		return err
+	}
+	return enc.EncodeString(req.expr)
```

Comment:
Just a code style issue, but please add a blank line after an error check.
```suggestion
	err := enc.EncodeMapLen(1)
	if err != nil {
		return err
	}

	err = enc.EncodeUint(uint64(iproto.IPROTO_SQL_TEXT))
	if err != nil {
		return err
	}

	return enc.EncodeString(req.expr)
```

Reply:
fixed